src/main/java/app/backend/controller/video/Detection.java

```diff
@@ -0,0 +1,54 @@
+package app.backend.controller.video;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class Detection {
+    private int id;
+    private int detectedCars;
+    private int detectedBuses;
+    private String connectionId;
+
+    public Detection() {
+    }
+
+    public static Detection[] getDetections(String jsonString) throws JsonProcessingException {
+
+        jsonString = jsonString.replaceAll("'", "\"");
+
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
zrób Autowire na ObjectMapperze jak np w CrossroadController

Reply:
to jest metda statyczna więc nie da się użyć w niej zautowirewanowangeo pola